### PR TITLE
Feature/create ingredients tests

### LINKED ISSUE #5: Build 'Ingredients' class test
- [x] Should have an id
- [x] Should have a name
- [x] Should have an estimated cost (in cents)
- [x] Should be able to calculate a cost by passing in quantity as an argument

### PR DESCRIPTION
**Pull request by mr-ryan12**
What (if any) features are you implementing?
- Added ingredients tests and a subset data file.

What (if anything) did you refactor?
- Nothing at the time.

Were there any issues that arose?
- Not at the time.

Is there anything that you need from your teammate?
- Pull request review.

Any other comments, questions, or concerns?
- Not at the moment

### CODE REVIEW

File: src/data/ingredients-test-data.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "name": "cheese",
+    "estimatedCostInCents": 5
+  },
+  {
+    "id": 3,
+    "name": "butter",
+    "estimatedCostInCents": 15
+  }
+];
+
```
Comment (philalewis) [inline src/data/ingredients-test-data.js:18]: Excellent list of ingredients for a grilled cheese sandwich

File: test/Ingredient-test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  });
+
+  it('should have an estimated cost in cents', () => {
+    expect(bread.estimatedCostInCents).to.equal(10);
+  });
+
+  it('should be able to calculate total cost', () => {
+    expect(bread.calculateCost(3)).to.equal(30);
+  });
+});
```
Comment (philalewis) [inline test/Ingredient-test.js:40]: Tests are the same as our bullet points on the project board. Excellent

**Review by philalewis — APPROVED** [review]
Looks good

File: test/Ingredient-test.js (hunk 2 above)
Comment (mr-ryan12) [reply test/Ingredient-test.js:40]: Thanks so much!

**Pull request merged**